lib/Creator/FeedCreator.php

```diff
@@ -12,44 +12,41 @@
     /**
      * Mandatory attributes of a feed.
      */
-    var $title, $description, $link;
-    var $format = 'BASE';
+    public $title, $description, $link;
+    public $format = 'BASE';
 
     /**
      * Optional attributes of a feed.
      */
-    var $syndicationURL, $image, $language, $copyright, $pubDate, $lastBuildDate, $editor, $editorEmail, $webmaster, $category, $docs, $ttl, $rating, $skipHours, $skipDays;
+    public $syndicationURL, $image, $language, $copyright, $pubDate, $lastBuildDate, $editor, $editorEmail, $webmaster, $category, $docs, $ttl, $rating, $skipHours, $skipDays;
 
     /**
      * The url of the external xsl stylesheet used to format the naked rss feed.
      * Ignored in the output when empty.
      */
-    var $xslStyleSheet = "";
+    public $xslStyleSheet = "";
 
 
-    /**
-     * @access private
-     */
-    var $items = Array();
+    /** @var FeedItem[] */
+    protected $items = Array();
```

Comment:
Any reason why this is `protected` instead of `public` as before? I think I might be using that property in one of my projects, so this change would break compat